modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -201,24 +200,31 @@ export default class Tile3DLayer extends CompositeLayer {
     const {gltf} = tileHeader.userData;
     const {instances, cartographicOrigin, modelMatrix} = tileHeader.content;
 
-    return new ScenegraphLayer({
-      id: `3d-model-tile-layer-${tileHeader.contentUri}`,
-      data: instances || [{}],
-      coordinateSystem: COORDINATE_SYSTEM.METER_OFFSETS,
-      coordinateOrigin: cartographicOrigin,
-
-      pickable: true,
-      scenegraph: gltf,
-      sizeScale: 1,
-      opacity: 1.0,
-      _lighting: 'pbr', // enable gltf pbr lighting model
-
-      // Fix for ScenegraphLayer.modelMatrix, under flag in deck 7.3 to avoid breaking existing code
-      _enableOffsetModelMatrix: true,
-      modelMatrix,
-      getTransformMatrix: instance => instance.modelMatrix,
-      getPosition: instance => [0, 0, 0]
-    });
+    const SubLayerClass = this.getSubLayerClass(
+      `3d-mode-tile-layer-${tileHeader.contentUri}`,
```

Comment:
I like the move to sublayer system, but I wonder if there is a limitation in that system for our use case:

It seems to me that we want the user to be able to override props for all pointcloud sublayers and all scenegraph sublayers, rather than individual, tile-URL dependent keys?

Or maybe this already works (since we do have the `TileLayer`). How would the user redefine all pointcloud layer props? 

@pessimistress

Reply:
No it doesn't... We need to make some changes to the sublayer props system.

Reply:
This can currently be worked around like this:

```js
const SubLayerClass = this.getSubLayerClass('3d-tile-layer-scenegraph', ScenegraphLayer);

new ScenegraphLayer(
  this.getSubLayerProps({
    id: '3d-tile-layer-scenegraph'
  }),
  {
    id: `3d-tile-layer-scenegraph-${tileHeader.contentUri}`,
    ...
  }
);
```

Reply:
In this way, the user is able to override the layer id
`id: '3d-tile-layer-scenegraph'`?

Reply:
That should work nicely, though I would add your comment explaining this (since it is non-standard usage of a somewhat obscure function):

//  lets use override all Scenegraph layer's using id '3d-tile-layer-scenegraph', but each sublayer will have a ${tileHeader.contentUri}

Also shouldn't the sublayer id string we match on just be `scenegraph`?

Reply:
Yes, it can be `${this.id}-scenegraph-${tileHeader.contentUri}`